pkg/networkplugin-syncer/handlers/ovn/router_ovn_cluster_north.go

```diff
@@ -0,0 +1,101 @@
+package ovn
+
+import (
+	goovn "github.com/ebay/go-ovn"
+	"github.com/pkg/errors"
+	"github.com/submariner-io/admiral/pkg/log"
+	"k8s.io/klog"
+)
+
+const (
+	// The ovn_cluster_router submariner port connects to the submariner router
+	ovnClusterSubmarinerRPort  = "ovn_cluster_subm_lrp"
+	ovnClusterSubmarinerSwPort = "ovn_cluster_subm_lsp"
+	ovnClusterSubmarinerMAC    = "00:60:2f:10:01:02"
+	ovnClusterSubmarinerNET    = ovnClusterSubmarinerIP + "/29"
+	ovnClusterSubmarinerIP     = "169.254.34.2"
```

Comment:
OVNKubernetes is already using 169.254.33.0/24 internally, it's a problem we will need to deal with.

I think it's better to use local link addresses here and follow the OVNKubernetes internal pattern.

I still want to find the way to remove that 224.0.0.0/8 which we used because we found no other way around, but we didn't have the time yet.